### PR TITLE
Add slash commands /query and /msg to match IRC

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/riot-web/issues/880

### CODE REVIEW

File: src/SlashCommands.tsx [hunk 1]
```diff
@@ -970,6 +971,51 @@ export const Commands = [
...
+        },
+        category: CommandCategories.actions,
+    }),
+    new Command({
+        command: "msg",
+        description: _td("Sends a message to the given user"),
+        args: "<user-id> <message>",
+        runFn: function(_, args) {
+            if (args) {
+                const matches = args.match(/^(\S+?)(?: +(.*))?$/s);
```
Comment (turt2live) [inline src/SlashCommands.tsx:998]: A comment to explain what we're looking for would be good. 

https://xkcd.com/1171/ etc

**Comment by turt2live** [comment]
also assuming product and friends still want this

**Comment by t3chguy** [comment]
It got put into project bite size so that seems clear

**Comment by t3chguy** [comment]
> assuming that regex isn't spawning some arcane magic in the 3rd century.

Its the same regex as we use for banning with a reason :D

**Pull request merged**